ensime_shared/ensime.py

```diff
@@ -833,23 +833,22 @@ def detect_row_column_start():
             row, col = self.cursor()
             start = col
             line = self.vim.current.line
-            # No completion from empty string
-            if start != 0 and line[start - 1] == " ":
-                start = -1
-            else:
-                while start > 0 and line[start - 1] != " ":
-                    start -= 1
-            return row, col, start + 1  # pos in no empty str
+            while start > 0 and line[start - 1] != " ":
```

Comment:
I forgot another common terminal case, the `.` operator. I think it probably makes sense for this condition to be:

```python
while start > 0 and (line[start -1] != " " and line[start - 1] != "."):
```

So we don't overwrite symbol names when chaining methods.